### ISSUE TITLE
[Bug]: Ars Nouveau Skyweave block invisible with shaders enabled

### ISSUE THREAD

**Issue by jp1224** [AllTheMods/ATM-9#1952]
### Possible Fixes

Yes

### Modpack Version

9-0.3.0

### What happened?

The Skyweave blocks appear invisible, showing the blocks around them, while any of the included shaderpacks are enabled. Turning shaders off lets the blocks show the skybox as intended.

**Comment by ThePhantomArc** [comment]
Yes This is still an issue to this day. If anyone has any ideas please help.
